README.md

```diff
@@ -3,9 +3,35 @@
 
 # OpenQA Public cloud Helper
 
-PublicCloud-Watcher or `pcw` is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs). It identifies abandoned instances by searching for certain tags and performs cleanup tasks. It also deletes old custom uploaded images. The behavior differs per CSP.
-
-The fastest way to run PublicCloud-Watcher is via the provided containers, as described in the [Running a container](#running-a-container) section.
+PublicCloud-Watcher (PCW) is a web app which monitors, displays and deletes resources on various Cloud Service Providers (CSPs).
+PCW has two main flows :
+
+1. Update run ( implemented in [ocw/lib/db.py](ocw/lib/db.py) )
+ Concentrates on deleting VMs (in case of Azure Resource Groups) . Each update run scans accounts defined in configuration file and
+and persists all results of each scan into sqlite database. If it is new entity than it obligatory gets TTL. TTL may be taken from tag `openqa_ttl` if
+entity is tagged with such tag if not PCW will check `pcw.ini` for `updaterun/default_ttl` setting if setting is not defined than PCW will use hard-coded value from [webui/settings.py](webui/settings.py). Database has a web UI where you can manually trigger certain entity deletion.
+
+ After persisting results into db PCW deciding which entities needs to be deleted. There are two ways to survive for entity :
```

Comment:
If the "Update" run also deletes CSP resources, I would actually rename it so "Seek for leftovers" or something similar. Update suggests no destructive means are taken place.

Reply:
sure make sense but I would go with function renaming too and move this into dedicated PR